editor/plugins/visual_shader_editor_plugin.cpp

```diff
@@ -3600,6 +3600,16 @@ void VisualShaderEditor::_setup_node(VisualShaderNode *p_node, const Vector<Vari
 			fmaFunc->set_op_type((VisualShaderNodeMultiplyAdd::OpType)(int)p_ops[0]);
 		}
 	}
+
+	// REMAP
+	{
+		VisualShaderNodeRemap *remapFunc = Object::cast_to<VisualShaderNodeRemap>(p_node);
```

Comment:
```suggestion
		VisualShaderNodeRemap *remap_func = Object::cast_to<VisualShaderNodeRemap>(p_node);
```
Please use `snake_case` for c++

Reply:
In this same function there are a bunch of other instances of camelCase, can I correct those at the same time?

Reply:
I'd leave those alone, and track them in:
* #91521